### ISSUE TITLE
Query regarding 'quantities-scidash' dependency

### ISSUE THREAD

**Issue by appukuttan-shailesh** [scidash/sciunit#201]
I am creating a Spack recipe for SciUnit, and I saw that it has a dependency for [quantities-scidash](https://pypi.org/project/quantities-scidash/). I see that this was added as a replacement for [quantities](https://pypi.org/project/quantities) in this [commit](https://github.com/scidash/sciunit/commit/7a11ca2c0b70193d5c66d5085cd6d14a2ead0195).

I am curious why we needed this change? If it indeed is different, can you point me to the GitHub (or similar) repo where this is maintained? The PyPI page for [quantities-scidash](https://pypi.org/project/quantities-scidash/) didn't seem to have this info (points to http://python-quantities.readthedocs.io/ which corresponds to the quantities package).

Spack already has the [quantities recipe](https://spack.readthedocs.io/en/latest/package_list.html#py-quantities) available, and so was wondering it is necessary to write a recipe for 'quantities-scidash' in order to use SciUnit.

**Comment by rgerkin** [comment]
@appukuttan-shailesh 
https://github.com/python-quantities/python-quantities/pull/186 was an important fix for sciunit compatibility and was taking a while to get integrated so I forked, applied the fix, and made a new version on pypi for the time being.  It looks like it got merged in in October so it should be in the v0.13.0 release of quantities which is now on pypi.  

If you'd like to submit a PR to switch back to quantities, and it passes all the unit tests, I'll merge it and make a new release.
